superduperdb/vector_search/in_memory.py

```diff
@@ -46,7 +46,10 @@ def __init__(
         self.identifier = identifier
 
     def __len__(self):
-        return self.h.shape[0]
+        if self.h:
+            return self.h.shape[0]
+        else:
+            return 0
```

Comment:
Please use `self.h is not None`, because this will raise an error when self.h is `numpy.array`

Reply:
yes nice catch, I have added it self.add but missed here